aten/src/ATen/record_function.h

```diff
@@ -213,6 +213,9 @@ struct TORCH_API RecordFunction {
       return;
     }
     state_->inputs_ = args.vec();
+    if (std::is_same<F, std::string>::value || std::is_same<F, const char*>::value) {
+      state_->op_input_size = state_->inputs_.size();
```

Comment:
@moderato Is it better to set these inside:
```
  void before(const char* name, int64_t sequence_nr = -1);
  void before(std::string name, int64_t sequence_nr = -1);
```
To remove these checks and be consistent with others?

Reply:
Done.